core/isolate.rs

```diff
@@ -50,11 +56,17 @@ pub enum StartupData<'a> {
   None,
 }
 
-type DispatchFn = Fn(&[u8], Option<PinnedBuf>) -> Op;
```

Comment:
This can be changed to:
```rust
type DispatchFn = Fn(bool, &[u8], Option<PinnedBuf>) -> Op<()>;
```

But no errors handling.

Reply:
My intent wasn't to handle them in core, but rather just to have a common function signature. That didn't work out since we need state as a context for dispatch in cli, so I will just revert this for now.